### ISSUE TITLE
Latest dev build ContentManager (Content.Load) throws an exception loading a skinned fbx model

### ISSUE THREAD

**Issue by YugeshN** [MonoGame/MonoGame#6211]
With version 3.7.0-1245, I get the following error when attempting to load a skinned FBX model:
{code}
An unhandled exception of type 'System.ArgumentException' occurred in mscorlib.dll An item with the same key has already been added.
{code}

Note, this is not a pipeline tool issue... this is loading the model at run-time using the following:
{code}
            Model model = Content.Load<Model>("models/dude/marine");
{code}

When I go back to version 3.7.0-1163, this code works perfectly and there are no issues, so it looks like the problem was introduced between version 3.7.0-1163 and version 3.7.0-1245

**Comment by tomspilman** [comment]
@YugeshN - Are you using `OpenAssetImporter` or `FbxImporter` for your model?   Try using `FbxImporter`.

**Comment by YugeshN** [comment]
@tomspilman  I am using the FbxImporter. I didn't realize there was the OpenAssetImporter in the pipeline tool. I can try that and see if it works... will need to re-install the newer (broken) dev build since I went back to the older (working) version.

**Comment by YugeshN** [comment]
@tomspilman 

You're right... the OpenAssetImporter is not working... I get the following exception with it:

```
Microsoft.Xna.Framework.Content.ContentLoadException
  HResult=0x80131500
  Message=Could not find ContentTypeReader Type. Please ensure the name of the Assembly that contains the Type matches the assembly in the full type name: Microsoft.Xna.Framework.Content.ReflectiveReader`1[[Microsoft.Xna.Framework.Content.Pipeline.Graphics.MaterialContent, MonoGame.Framework.Content.Pipeline, Version=3.7.0.1245, Culture=neutral, PublicKeyToken=null]] (Microsoft.Xna.Framework.Content.ReflectiveReader`1[[Microsoft.Xna.Framework.Content.Pipeline.Graphics.MaterialContent, MonoGame.Framework.Content.Pipeline]])
  Source=MonoGame.Framework

```

So it doesn't look like loading the model works with either fbx or openasset importer in the new build.

However, it looks like both fbximport and openassetimporter work in the older 1163 build. So something broken along the way.

**Comment by tomspilman** [comment]
Actually... this might be this bug which was fixed a few days ago:  https://github.com/MonoGame/MonoGame/pull/6191

Try the latest development build.  I suspect it will fix you.
